test/explicit-resource-management/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "explicit-resource-management",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "run": "npm run build && node out/main.js",
+    "build": "tsc",
+    "test": "npm run build && mocha --config .mocharc.json main.test.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@types/chai": "^4.3.16",
+    "chai": "^4.4.1",
+    "mongodb": "../../mongodb-6.8.0.tgz",
```

Comment:
I wanted the smoke test to use a packaged driver, this was intentional

Reply:
Sure, and fine to leave it as is, for my understanding What is or Is there a difference? what is a package testing that a symlink doesn't? IIUC npm prepare is the step that runs in either case, is there something else?

Reply:
I wanted it to be as close as possible to what users will experience, that's all.  Ideally we'd install a real beta driver to test with, but since we can't do that per-CI run, npm pack was the next best option.

Reply:
What's wrong with npm pack?

Reply:
> What's wrong with npm pack?

Nothing :) and I'll resolve this as it was a suggestion anyway